package.json

```diff
@@ -68,100 +68,104 @@
   },
   "homepage": "https://github.com/concord-consortium/question-interactives#readme",
   "devDependencies": {
-    "@cypress/webpack-preprocessor": "^5.4.11",
-    "@svgr/webpack": "^5.4.0",
-    "@testing-library/jest-dom": "^5.14.1",
+    "@cypress/webpack-preprocessor": "^5.11.1",
+    "@svgr/webpack": "^6.2.1",
+    "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^10.4.9",
-    "@testing-library/react-hooks": "^3.4.1",
-    "@types/chart.js": "^2.9.28",
+    "@testing-library/react-hooks": "^3.7.0",
+    "@types/chart.js": "^2.9.35",
     "@types/deep-equal": "^1.0.1",
-    "@types/dompurify": "^2.0.3",
-    "@types/enzyme": "^3.10.5",
+    "@types/dompurify": "^2.3.3",
+    "@types/enzyme": "^3.10.11",
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/jest": "^26.0.10",
-    "@types/object-hash": "^2.1.1",
-    "@types/react": "^16.9.48",
-    "@types/react-dom": "^16.9.8",
-    "@types/react-jsonschema-form": "^1.7.4",
-    "@types/uuid": "^8.3.0",
-    "@types/video.js": "^7.3.10",
-    "@typescript-eslint/eslint-plugin": "^3.10.1",
-    "@typescript-eslint/parser": "^3.10.1",
-    "autoprefixer": "^9.8.6",
+    "@types/jest": "^26.0.24",
+    "@types/object-hash": "^2.2.1",
+    "@types/react": "^16.14.23",
+    "@types/react-dom": "^16.9.14",
+    "@types/react-jsonschema-form": "^1.7.8",
+    "@types/uuid": "^8.3.4",
+    "@types/video.js": "^7.3.31",
+    "@typescript-eslint/eslint-plugin": "^5.13.0",
+    "@typescript-eslint/parser": "^5.13.0",
+    "autoprefixer": "^9.8.8",
     "babel-core": "^6.26.3",
-    "babel-jest": "^26.3.0",
-    "babel-loader": "^8.1.0",
+    "babel-jest": "^26.6.3",
+    "babel-loader": "^8.2.3",
     "canvas": "2.6.1",
-    "css-loader": "^4.2.2",
-    "cypress": "^6.0.0",
+    "css-loader": "^4.3.0",
+    "cypress": "^9.5.1",
     "enzyme": "^3.11.0",
```

Comment:
You replaced `enzyme` with `@testing-library` in at least one instance below. Is `enzyme` still used elsewhere or could it be removed at this point?

Reply:
@kswenson  Enzyme is still required by a bunch of tests that use its `mount` and `shallow`. I guess we could change those to use React Testing Library instead, but I think that's probably beyond the scope of this work -- unless you think it's important?

Reply:
In the long run I think that should be the plan, but it's not a priority for this PR. I just saw that you changed it in one place and wanted to make sure that it hadn't become a stale dependency.